packages/adapter-auto/package.json

```diff
@@ -16,8 +16,6 @@
 		},
 		"./package.json": "./package.json"
 	},
-	"main": "index.js",
-	"types": "index.d.ts",
```

Comment:
Does publint warn about the `types` field? I think we should still keep this since none-node16 `moduleResolution` projects wouldn't read from the `exports` field `types` condition, so you'd still have to keep it. (Last I checked)

Reply:
no, it doesn't. I assumed it was also superfluous. I've added it back